Tasks/JavaToolInstallerV0/javatoolinstaller.ts

```diff
@@ -81,6 +73,22 @@ async function getJava(versionSpec: string): Promise<void> {
     toolLib.prependPath(path.join(jdkDirectory, BIN_FOLDER));
 }
 
+/**
+ * Delete the contents of the destination directory but leave the directory in place
+ * @param directory Directory path
+ */
+function cleanFolder(directory: string) {
+    // Clean the destination folder before downloading and extracting
+    if (taskLib.exist(directory) && taskLib.stats(directory).isDirectory) {
+        console.log(taskLib.loc('CleanDestDir', directory));
+        fs.readdirSync(directory)
+            .forEach((item: string) => {
+                const itemPath = path.join(directory, item);
+                taskLib.rmRF(itemPath);
```

Comment:
I assume it will be better to put all extracted logic in a `try..catch` block. Also, this function can return a bool value which will indicate is cleaning was successful or not.